x-pack/plugins/security_solution/public/management/pages/index.tsx

```diff
@@ -16,13 +18,54 @@ import {
 import { NotFoundPage } from '../../app/404';
 import { EndpointsContainer } from './endpoint_hosts';
 import { getEndpointListPath } from '../common/routing';
+import { SpyRoute } from '../../common/utils/route/spy_routes';
+import { SecurityPageName } from '../../app/types';
+import { useIngestEnabledCheck } from '../../common/hooks/endpoint/ingest_enabled';
+
+const NoPermissions = memo(() => {
+  return (
+    <>
+      <EuiEmptyPrompt
+        iconType="alert"
+        iconColor="danger"
+        titleSize="l"
+        data-test-subj="noIngestPermissions"
+        title={
+          <FormattedMessage
+            id="xpack.securitySolution.endpointManagemnet.noPermissionsText"
+            defaultMessage="You do not have the required Kibana permissions to use Elastic Security Administration"
+          />
+        }
+        body={
+          <p>
+            <EuiText color="subdued">
+              <FormattedMessage
+                id="xpack.securitySolution.endpointManagemnet.noPermissionsSubText"
+                defaultMessage="It looks like Ingest Manager is disabled. Ingest Manager must be enabled to use this feature. If you do not have permissions to enable Ingest Manager, contact your Kibana administrator."
+              />
+            </EuiText>
+          </p>
+        }
+      />
+      <SpyRoute pageName={SecurityPageName.management} />
+    </>
+  );
+});
+NoPermissions.displayName = 'NoPermissions';
 
 export const ManagementContainer = memo(() => {
   const history = useHistory();
+  const { allEnabled: isIngestEnabled } = useIngestEnabledCheck();
   return (
     <Switch>
-      <Route path={MANAGEMENT_ROUTING_ENDPOINTS_PATH} component={EndpointsContainer} />
-      <Route path={MANAGEMENT_ROUTING_POLICIES_PATH} component={PolicyContainer} />
+      <Route
```

Comment:
I had been wondering about this. You had mentioned before (maybe standup) that the tab was disabled, but rally, what is happening is that we still show/enable the tab, but display a different page - correct?

Also - Instead of using the conditional below in all `<Route>`'s, just add one at before this `return`, so that we don't have to keep writing the condition on each `<Route>` - like : 

```jsx
  const { allEnabled: isIngestEnabled } = useIngestEnabledCheck();
  
  if (!isIngestEnabled) {
     return `<Route path="*" component={NoPermissions} />`;
  }

  return (
    <Switch>
      <Route path={MANAGEMENT_ROUTING_ENDPOINTS_PATH} component={EndpointsContainer} />
      <Route path={MANAGEMENT_ROUTING_POLICIES_PATH} component={PolicyContainer} />
      //....
```

Reply:
@paul-tavares yes, we do want the tab to still show up, that's an ask from product so that we do not hide newer features.  This way users can discover why they are not able to use the feature and adjust permissions accordingly.

Reply:
Ok - The issue is on how the mock is defined then. It probably just needs to be adjusted there to something like `jest.Mock<Typehere>`. 
We can address it later (maybe open an issue so we don't forget :) )